tests/qnn/test_cost.py

```diff
@@ -91,3 +91,12 @@ def test_rx_circuit(self, interface):
         res = loss(phis, target=np.array([1.0, 0.5, 0.1]))
 
         assert np.allclose(res, np.array([0.21, 0.25, 0.03]), atol=0.01, rtol=0.01)
+
+
+def test_deprecation_warning():
```

Comment:
Nice :+1: 

I wonder if we can set pytest to automatically fail if this test is run on PennyLane version > 0.15, so that we remember to make the change

Reply:
That's a good idea! I'll add that.

Reply:
Added now: 108c6c097ecf709070521cb730d2773d824932f5